### ISSUE TITLE
qtwebkit-kiosk/server: turn display off/on

### ISSUE THREAD

**Issue by jpkorhonen** [opinsys/iivari#6]
Configurations by organisation (config/organisation.yml)
- Days of the week (display is on)
- Start time (turn display on)
- End time (turn display off)

The server transmits the settings to the client in the same api as, for example, the slides data.

qtwebkit-kiosk software turns the display on or off with the following shell command:

``` bash
#!/bin/sh
# turn info-tv display off
export DISPLAY=:0.0
xset dpms force standby
```

``` bash
#!/bin/sh
# turn info-tv display on
export DISPLAY=:0.0
xset s reset
xset s 0 0
xset s off
xset s noblank
xset dpms 0 0 0
xset -dpms
xset dpms force on
```

**Comment by lamikae** [comment]
Python backend can receive signals from JavaScript. See commit 4dcbdbc4b7863f07dc70efb6ba706b13a2433f52.

Signals can be triggered through window.displayCtrl object (written in CoffeeScript).

Implementing powerup and -down signals requires some work with the JSON structure.

**Comment by lamikae** [comment]
Implemented in commits 497c6143a78c4eeb225a1a95da44a402954272ec and 5bfcdf90f091ac0e628c45bd4cc9b25ef1c46d27.

The configuration is shown in [organisations.yml.example](http://github.com/opinsys/iivari/blob/lamikae/server/config/organisations.yml.example)

**Comment by lamikae** [comment]
Ported to develop branch in 192db9e50483239eaeee825b6bb62e046841b87e
